ramls/records-editor.raml

```diff
@@ -0,0 +1,72 @@
+#%RAML 1.0
+
+title: QuickMARC Editor Records
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://github.com/folio-org/mod-quck-marc
+
+documentation:
+  - title: QuickMARC record API
+    content: This documents the API calls that can be made to work with MARC records
+
+types:
+  marcUi: !include schemas/marc_ui.json
+  errors: !include raml-storage/raml-util/schemas/errors.schema
+  UUID:
+    type: string
+    pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+  validate: !include raml-storage/raml-util/traits/validation.raml
+
+/records-editor/records:
+  get:
+    queryParameters:
+      /^(id|instanceId)$/: UUID
+    responses:
+      200:
+        description: "Returns an item with a given ID"
+        body:
+          application/json:
+            type: marcUi
+            example:
+              strict: false
+              value: !include examples/marc_ui.sample
+      404:
+        description: "Item with a given ID not found"
+        body:
+          text/plain:
+            example: "MARC record not found"
+      500:
+        description: "Internal server error, e.g. due to misconfiguration"
+        body:
+          text/plain:
+            example: "Internal server error, contact administrator"
+  /{id}:
+    put:
```

Comment:
Looks like smth goes wrong. I don't see signature for PUT method after interface generation in the resource target/generated-sources/raml-jaxrs/org/folio/rest/jaxrs/resource/RecordsEditorRecords.java. Could you please doublecheck raml-file.

Reply:
Fixed

Reply:
Fixed